bigquery/google/cloud/bigquery/schema.py

```diff
@@ -14,7 +14,7 @@
 
 """Schemas for BigQuery tables / queries."""
 
-import collections
+from collections import abc
```

Comment:
```suggestion
try:
    from collections.abc import Mapping
except ImportError:
    # python 2.7
    from collections import Mapping
```

Reply:
It's not that I "like" it, it's that this way it's easy to tell what module something came from.

We should use the `six` module, if the `abc` module is not available on 2.7.